drivers/serial/uart_b9x.c

```diff
@@ -187,9 +187,13 @@ static void uart_b9x_cal_div_and_bwpc(uint32_t baudrate, uint32_t pclk,
 static void uart_b9x_init(volatile struct uart_b9x_t *uart, uint16_t divider,
 			  uint8_t bwpc, uint8_t parity, uint8_t stop_bit)
 {
+#if CONFIG_SOC_RISCV_TELINK_B95
+	uart->ctrl0 = ((uart->ctrl0 & (~FLD_UART_BPWC_O)) | bwpc);
+#else
+	uart->ctrl0 = bwpc;
+#endif
```

Comment:
same here lets use one style for #if SoCs as above
and what is the reason to move this upper line "divider = ..." ?

Reply:
actually it's meaning less , but i think if need ,we shoud move the like that(related code works together)
/* config clock */
	divider = divider | FLD_UART_CLK_DIV_EN;